axelrod/strategies/finite_state_machines.py

```diff
@@ -54,8 +54,9 @@ def __init__(self, transitions=None, initial_state=None, initial_action=None):
             initial_state = 1
             initial_action = C
         Player.__init__(self)
+        self.initial_state = initial_state
         self.initial_action = initial_action
-        self.fsm = SimpleFSM(transitions, initial_state)
+        self.fsm = SimpleFSM(transitions, self.initial_state)
```

Comment:
Nope, no difference. Not sure why I did that, I would actually argue the style before was better: changing now and cleaning up the commit history too :)